csrc/host_ir/jit.cpp

```diff
@@ -166,6 +180,536 @@ void printLlvmIr(llvm::Function* func, std::string_view msg) {
   llvm::outs() << "\n\n";
 }
 
+// Helper function to traverse the extent of a Val and generate LLVM IR
```

Comment:
The current comment doesn't give more information than the function name. Consider removing it or adding something more useful.